### PR TITLE
Login sync page

### PR DESCRIPTION
**Pull request by TiannaW24**
Basic page that doesn't do much yet. Pulling in the USWDS style library that we were shown.

### CODE REVIEW

File: webportal/webapp/package.json [hunk 1]
```diff
@@ -36,6 +36,7 @@
     "ngx-md": "^3.1.0",
     "primeng": "^5.2.7",
     "rxjs": "^5.5.4",
+    "uswds": "^1.6.13",
```
Comment (TiannaW24) [inline webportal/webapp/package.json]: Pull in new style library

File: webportal/webapp/src/assets/js/loginsync.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+function togglePassword(e1, e2) {
```
Comment (TiannaW24) [inline webportal/webapp/src/assets/js/loginsync.js]: Question: I noticed this webapp doesn't have a lot of Javascript. Wondering if that's a pattern or a coincidence.

File: webportal/webapp/src/app/account/loginsync/togglepassword.directive.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import { Directive, ElementRef } from '@angular/core';
+
+@Directive({
```
Comment (TiannaW24) [inline webportal/webapp/src/app/account/loginsync/togglepassword.directive.ts]: Still getting the lay of the land, let me know if this should live somewhere else.

File: webportal/webapp/src/app/account/loginsync/loginsync.component.html [hunk 4]
```diff
@@ -0,0 +1,19 @@
+<div class="container">
+  <mat-card>
+    <div>
+      <form class="usa-form signin" (ngSubmit)="onSubmit()">
+        <fieldset class="usa-fieldset">
+          <legend class="usa-legend">Sign in</legend>
+  
+          <label for="username">Username or email address</label>
```
Comment (aaronFranssell) [inline webportal/webapp/src/app/account/loginsync/loginsync.component.html]: This will just be a username, not an email address.

Comment (TiannaW24) [reply webportal/webapp/src/app/account/loginsync/loginsync.component.html]: kk

File: webportal/webapp/src/app/account/loginsync/loginsync.component.html [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+          <input class="usa-input" id="username" name="username" [(ngModel)]="username" type="email" autocapitalize="off" autocorrect="off">
+  
+          <label for="password-sign-in">Password</label>
+          <input class="usa-input" id="password-sign-in" name="password" [(ngModel)]="password" type="password" appTogglePassword>
```
Comment (aaronFranssell) [inline webportal/webapp/src/app/account/loginsync/loginsync.component.html]: Why are we showing the password?

Comment (TiannaW24) [reply webportal/webapp/src/app/account/loginsync/loginsync.component.html]: you know how sometimes you're not sure if you typed your password correctly and you wanna peak at it? Thats what that is for.

File: webportal/webapp/src/app/account/loginsync/loginsync.component.html [hunk 6]
```diff
@@ -0,0 +1,19 @@
+<div class="container">
+  <mat-card>
+    <div>
+      <form class="usa-form signin" (ngSubmit)="onSubmit()">
+        <fieldset class="usa-fieldset">
+          <legend class="usa-legend">Sign in</legend>
+  
+          <label for="username">Username</label>
+          <input class="usa-input" id="username" name="username" [(ngModel)]="username" type="email" autocapitalize="off" autocorrect="off">
```
Comment (aaronFranssell) [inline webportal/webapp/src/app/account/loginsync/loginsync.component.html]: Can you remove the other email specific types?

Comment (TiannaW24) [reply webportal/webapp/src/app/account/loginsync/loginsync.component.html]: woops, nice catch. 👍

**Pull request merged**